### ISSUE TITLE
Readme example does not seem to work

### ISSUE THREAD

**Issue by pixelastic** [textlint-rule/textlint-rule-alex#1]
Hello,

I just installed `textlint` along with this `textlint-rule-alex` set of rules. I copy-pasted the `.textlintrc` of the example and tested on a very simple file. Still, the rule is not ignored.

```
$ cat ~/.textlintrc
{
  "rules": {
    "alex": {
      "allow": ["boogeyman-boogeywoman"]
    }
  }
}
```

```
$ cat textlint.md
Are you a boogeyman?
```

```
$ textlint textlint.md

/home/tim/local/tmp/textlint.md
  1:11  error  [boogeyman-boogeywoman] `boogeyman` may be insensitive, use `boogey` instead  alex

✖ 1 problem (1 error, 0 warnings)
```

```
$ textlint --version
v7.0.1
```

I tried with several rules, but could not manage to ignore any of them. Am I doing something wrong?

**Comment by azu** [comment]
@pixelastic Thanks for report.
I fix it and release textlint-rule-alex@1.1.0

**Comment by pixelastic** [comment]
Thanks, working well now.
